src/sciline/utils.py

```diff
@@ -16,3 +18,13 @@ def qualname(obj: Any) -> str:
     return str(
         obj.__qualname__ if hasattr(obj, '__qualname__') else obj.__class__.__qualname__
     )
+
+
+def keyname(key: Key) -> str:
+    if isinstance(key, Item):
+        return f'{keyname(key.tp)}({keyname(key.label[0].tp)})'
+    args = get_args(key)
+    if len(args):
+        parameters = ', '.join(map(keyname, args))
+        return f'{qualname(key)}[{parameters}]'
+    return qualname(key)
```

Comment:
How does this compare to what is used in `visualize`? Can we use the same code?

Reply:
Yes I think we can.